examples/src/test/scripts/org/kaazing/k3po/examples/internal/server.hello.world.rpt

```diff
@@ -14,9 +14,11 @@
 # limitations under the License.
 #
 
-accept tcp://localhost:8001
+property location "tcp://localhost:8001"
+property RESPONSE "hello client"
```

Comment:
How about including a third property that is not overridden? Just to make sure all the properties don't get replaced.